scripts/variantstore/wdl/GvsCallsetCost.wdl

```diff
@@ -75,29 +78,93 @@ task WorkflowComputeCosts {
     }
 }
 
-#task BigQueryWriteAPICost {
-#    meta {
-#        description: "Estimate GvsImportGenomes use of the BQ Write API via core storage costs from the sizes of vet_% and ref_ranges_% tables."
-#        volatile: true
-#    }
-#
-#    input {
-#        String project_id
-#        String dataset_name
-#    }
-#    command <<<
-#    >>>
-#
-#    runtime {
-#        docker: ""
-#    }
-#
-#    output {
-#        Float vet_gib = read_float("")
-#        Float ref_ranges_gib = read_float("")
-#        Float import_genomes_cost = 3
-#    }
-#}
+task CoreStorageModelSizes {
+    input {
+        String project_id
+        String dataset_name
+    }
+    meta {
+        description: "Read sizes of vet_%, ref_ranges_%, and alt_allele tables from `INFORMATION_SCHEMA.PARTITIONS`."
+        # Definitely don't cache this, the values will change while the inputs to this task will not!
+        volatile: true
+    }
+    command <<<
+
+        sanity_check_project() {
```

Comment:
Why validate the project id?  And if there are good reasons to validate it, why just here?

Reply:
This gets interpolated into various BQ queries and given all the recent Hack EDU training I thought a little validation might be in order. 🙂  I was working on centralizing all input validation to one task in my next PR with downstream tasks only running if that succeeds.

Reply:
actually since I have reviewers' attention now I'll just mosey that code on from my in-progress branch to this one... 🙂